draft-irtf-cfrg-randomness-improvements.md

```diff
@@ -155,13 +159,13 @@ is available and CSPRNG randomness guarantees are dubious, or to provide stronge
 about possible future issues with the randomness. Roughly, the security properties provided
 by the proposed construction are as follows:
 
-1. If the CSPRNG works fine, that is, in a certain adversary model the CSPRNG output is
+1. If the CSPRNG works fine, that is, in a certain adversarial model the CSPRNG output is
```

Comment:
I wouldn't change "adversary model" to "adversarial model"; "adversary model" seems to be a more common term